### ISSUE TITLE
All Events Cost Center Totals

### ISSUE THREAD

**Issue by fishbaugher** [fishbaugher/WAweb-Issues#6]
Breakdown of Event Type, Cost Center Type, Cost Center Name, Expenditures, and Units Using Funds.

A critical piece of business logic needs to be detailed:

What exactly constitutes a unit of completion, and when is the unit of completion credited?

Report per:  Agency or all Agencies for state
Existing ReportID: 125
Generated by:  Agency or State

**Comment by fishbaugher** [comment]
Here is a sample report for the SRC agency for PY23 to date production:

[Report125.pdf](https://github.com/fishbaugher/WAweb-Issues/files/13891976/Report125.pdf)

**Comment by gwyttenback** [comment]
A standalone unit is complete when it passes QCI.  A WX audit though is not completed until the audit is completed and locked on.

**Comment by gwyttenback** [comment]
Yes, I agree with everything stated below for standalone and audit.

We do not use response to client request and call-backs are not completed units as they tie back to an original job that was once counted as completed.
